react/components/GalleryItem.js

```diff
@@ -9,8 +10,8 @@ import { PropTypes } from 'prop-types'
 /**
  * Normalizes the item received in the props to adapt to the extension point prop.
  */
-const GalleryItem = ({ item, displayMode, summary }) => {
-  const normalizeProductSummary = product => {
+class GalleryItem extends Component {
```

Comment:
why did you move it to a class component?

Reply:
Because the Pixel "interface" should have access to a component's props to push notifications.

Reply:
I will follow the suggestion of @jgfidelis and use usePixel. Didn't know its existance. :)

Reply:
by the way, you can still use high order components in function components, its the same as you would use in a class. there was no need to convert it into a class even if the `usePixel` didn't exist

Reply:
check [this guide](https://reactjs.org/docs/higher-order-components.html) if you want to learn more